python/nano/test/pytorch/tests/inference/test_bf16_ipex.py

```diff
@@ -151,6 +152,116 @@ def hello():
             new_model.width
 
 
+    def test_bf16_channels_last_inference_with_jit(self):
+        model = resnet18(num_classes=10)
+
+        x = torch.rand((10, 3, 256, 256))
```

Comment:
it seems we keep using `x=torch.rand((10, 3, 256, 256))` as the test input sample. An easy way to do this could be
1. add 1 test function in each test file (test_bf16, test_bf16_ipex, test_ipex_jit_inference)
2. For these test function, write your own test model (could be an extremely easy one with 3 input)
3. use (x1, x2, x3) as input, where x1, x3 are 4-dim tensor while x2 is 3-dim.
4. test save/load